themes/navy/source/css/_partial/index.styl

```
@@ -178,7 +178,7 @@
   border-color: color-border
   &:hover
     background: @border-color
-    color: #283542
+    color: color-default
```

```suggestion
    color: lighten(color-navy, 10%)
```

The text color on hover should be the same as the background color `#theme-wrap` has.